Library/Homebrew/cli/named_args.rb

```diff
@@ -106,7 +106,7 @@ def to_resolved_formulae_to_casks(only: nil)
       def to_objects(only: nil, method: nil)
         @to_objects ||= {}
         @to_objects[only] ||= downcased_unique_named.flat_map do |name|
-          next Tap.fetch(name) if only == :tap || (only.nil? && name.count("/") == 1 && !name.start_with?("./", "/"))
+          next Tap.fetch(name) if only == :tap
```

Comment:
I'm not sure if we ever actually hit this with `only == :tap`.

Reply:
@reitermarkus Reckon we can just remove this line altogether?